etc/config-private.pl

```diff
@@ -165,6 +165,16 @@
     #     username => '...',
     #     password => '...',
     # );
+
+    # Configuration of BlobStore. This is the new storage abstraction used to
+    # store any blobs (images, userpics, media, etc) that need storage. For small
+    # sites/single servers, the localdisk mode is useful. For production
+    # systems S3 should be used.
+    # %BLOBSTORE = (
+    #     localdisk => {
+    #         path => "$LJ::HOME/var/blobstore",
+    #     },
+    # );
```

Comment:
I don't think so?

* config.pl - stuff that is useful to every site
* config-local.pl - stuff that customizes the site to DW, but isn't production related; i.e., a Hack is made to 'behave just like DW' by sharing the same config-local.pl
* config-private.pl - stuff that is particular to a running version of the site like DB credentials, IPs, paths, etc.